packages/components/src/shared/constants.ts

```diff
@@ -5,6 +5,7 @@ export const DEFAULT_MESSAGE_ID_SUFFIX: string = '-message';
 export const DEFAULT_VALID_MESSAGE_ID_SUFFIX: string = '-valid-message';
 export const DEFAULT_INVALID_MESSAGE_ID_SUFFIX: string = '-invalid-message';
 export const DEFAULT_PLACEHOLDER_ID_SUFFIX: string = '-placeholder';
+export const DEFAULT_DATALIST_ID_SUFFIX: string = '-placeholder';
```

Comment:
```suggestion
export const DEFAULT_DATALIST_ID_SUFFIX: string = '-datalist';
```

my understanding is, that this could be `datalist` instead of `placeholder` for this one, couldn't it ?